back_end/index.js

```diff
@@ -419,6 +419,163 @@ app.get('/fornecedor/:id', (req, res) => {
   console.log('get /fornecedor/' + fornecedorId + ' executado. Fornecedor retornado com sucesso!');
 });
   
+
+//alteração Erik e Marcos//
+
+app.get('/usuario', (req, res) => {
+  db.query('SELECT * FROM usuario', (err, results) => {
```

Comment:
É pressuposto que se retorne todos os usuários, incluindo as senhas em hash salvas nas colunas de cada usuário dentro do banco?

Reply:
poderia remover o campo senha do retorno, mas eu deixaria assim mesmo

Reply:
> É pressuposto que se retorne todos os usuários, incluindo as senhas em hash salvas nas colunas de cada usuário dentro do banco?

O campo de privilégio vai ter no retorno pra exibir no front-end quais o usuário possui?

Reply:
privilegio precisa ter no front pra fazer validações do lado dele, o zé falou disso pelo q eu lembro

Reply:
Acho que não é seguro devolver o objeto do aluno com a senha pro front, pq se esse objeto for interceptado, a senha estará bem ali, embora esteja com hash, não implementamos nenhum meio de criptografia entre os pacotes, portanto acredito que seja melhor prevenir do que remediar, tbm ganhar uns pontinhos com o avaliador se ele for detalhista rs

Reply:
vai ficar assim mesmo retornando a senha criptografada